railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb

```diff
@@ -5,11 +5,23 @@
 <% module_namespacing do -%>
 class <%= controller_class_name %>Controller < ApplicationController
   before_action :set_<%= singular_table_name %>, only: [:show, :edit, :update, :destroy]
+  
+  <%- if collection_routing? -%>
+  # GET <%= route_url %>/1..10,17
+  def index
+    @<%= plural_table_name %> = <%= singular_table_name %>_ids ? <%= orm_class.find(class_name, "#{singular_table_name}_ids") %> : <%= orm_class.all(class_name) %>
```

Comment:
I agree with you. Instead of changing `post_ids` to return a boolean we should use `post_ids.any?`. I think it also reads better.